.github/workflows/build.sh

```diff
@@ -50,8 +50,7 @@ function build() {
   git clone --depth 1 --branch openssl-3.1.3 https://github.com/openssl/openssl.git
   pushd openssl
   ./Configure $OPENSSL_ARCH --prefix=$deps
-  make -j$(nproc)
-  make -j$(nproc) install_sw
+  make -j$CMAKE_BUILD_PARALLEL_LEVEL install_sw
```

Comment:
This doesn't actually work!

You can't even do `build_sw install_sw` in the same step. Super weird! (and probably a bug)